README.md

```diff
@@ -1,12 +1,40 @@
-# Repo for SRE Content
+# Operate First SRE Content
 
-##  Material
+## Introduction
 
-* [Architecture Decision Records](./ADRs/)
-* [SRE Maturity Model](./sre_maturity.md)
-* [Picking Good SLIs](./picking_good_slis.md)
+This the Operate First repository for Site Reliability documentation. Here we
+intend to collect documentation on the nitty-gritty aspects of building and
+operating an effective Site Reliability organisation. 
+
+## Who runs this repository?
+
+Hello! We're the Red Hat SIG-SRE group, a cross-team group of people inside Red
+Hat with an interest in promoting and developing strong SRE culture. As a
+result you can expect some of the material here to be a little slanted toward
+the way we do things here inside Red Hat, but as we're doing our best to stick
+to what are generally accepted to be good SRE practices you should be able to
+make use of it with, at most, minor modification.
+
+##  What's available here?
+
+### Service Metrics
+
+* [Picking good Service Level Indicators (SLIs)](./picking_good_slis.md)
+* [Picking good Service Level Objectives (SLOs)](./picking_good_slos.md)
+
+### Decision-Making
+
+* Some sample SIG-SRE [Architecture Decision Records](./ADRs/RH/SIG-SRE)
+
+### Other Documents
+
+* A work in progess - the [SRE Maturity Model](./sre_maturity.md)
 
 ## Contributing
 
-Please check for any markdown linting problems.
-The [vscode-markdownlint](https://github.com/DavidAnson/vscode-markdownlint) plugin can be used to do this if using vscode.
+Feedback is always welcome, and contributions are too! Feel free to file an
+issue, or simply send us a PR. 
```

Comment:
Here could be a place to point to where synchronous communications could occur.  Link to op1st community landing page?  @quaid if you agree, what would be the ideal hyperlink to put here?

Reply:
Agree about a link to a landing page, https://www.operate-first.cloud/our-community is canonical.

I'm on my phone app and it won't let me make a markdown suggestion, so two other points on this line:
- the word issue should be a link to the repository issue page, and even to a specific template if that's more useful
- the common technical writer recommendation is to remove words such as "simply" because the size of a portion of people will silently segfault at words like that in documentation when it doesn't align with their feeling or experience
- and as with all acronyms, you always spell it out in the first usage and supply the acronym in parentheses, thus: "you can make a pull request (PR) or ..."
- bonus would be linking the words "make a pull request" to our canonical page on how to make a pull request for the project that includes GitHub basics doc links, when that page exists ... :(

If it helps, another example is the word "just", as in, "just make a pull request". ;)

Reply:
Thanks - I've folded those in, and good catch on the "simply".